experimental/Schemes/SpaceGerms.jl

```diff
@@ -1,9 +1,16 @@
+export HypersurfaceGerm
+export CompleteIntersectionGerm
 export SpaceGerm
 export ambient_germ
 export germ_at_point
+export hypersurface_germ
+export complete_intersection_germ
+export is_isolated_singularity
 export point
 export rational_point_coordinates
 export representative
+export milnor_algebra
+export milnor_number
```

Comment:
Export statements should now be moved to `src/exports.jl` (or something similar).

Reply:
done